docs/src/create-apps/workers.md

```diff
@@ -3,7 +3,7 @@ title: "Work with workers"
 description: Interact with your worker instances to handle background tasks for your apps.
 ---
 
-Workers are instances of your code that can't interact with the outside world.
+Workers are instances of your code that aren't open to connections with other apps or services or the outside world.
```

Comment:
You can still connect outbound from worker -> external app, I think you mean that they cannot connect inbound external app -> worker

```suggestion
Workers are instances of your code that aren't open to connections from other apps or services or the outside world.
```

Reply:
I think that's what I had in my head, but my fingers didn't seem to agree.